src/editor/executor.ts

```diff
@@ -110,7 +110,13 @@ export class ExecutorAlerts {
             break;
         case ProcessStatus.errored:
             this.statusBarItem.text = '$(testing-error-icon) CodeChecker: analysis errored';
-            window.showErrorMessage('CodeChecker finished with error - see logs for details');
+
+            const shouldShowError = workspace.getConfiguration('codechecker.executor')
```

Comment:
I wouldn't hide Codechecker runtime error notifications in the case when the user ticks the newly introduced option. The settings description says: "Enable notifications when CodeChecker is not found or not supported", but I suspect there can be other errors too. Partial/total suppression of notifications could be introduced, like with a slider or with another tickbox, that says, "Disable every popup originating from the plugin" or something along the way.

If the cases for a failed analysis and other runtime errors are handled elsewhere than this, please consider the above remarks irrelevant.

Reply:
Changed the option to `Enable CodeChecker-related notifications`, as other notifications being spammed has also been an issue before. It now controls every notification, except when a report cannot be found when trying to jump to it.

I'm planning to show the notification's contents in the sidebar, but that will be added in another PR.